src/components/DocumentRenderer/ValidityBanner/ValidityBannerHeader.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { FunctionComponent, useState, useEffect } from "react";
+import { View, Text } from "react-native";
+import { RectButton } from "react-native-gesture-handler";
+import { Feather } from "@expo/vector-icons";
+import { CheckStatus } from "./types";
+import { ValidityIcon } from "./ValidityIcon";
+
+interface ValidityBannerHeader {
+  checkStatus?: CheckStatus;
+  isExpanded?: boolean;
+  progress?: number;
```

Comment:
Should the progress be calculated in this component instead? With the info on the CheckStatus, it seemed like it can be determined from there.

Reply:
Hmm, since the header only knows the overall check status, it doesn't know the status of the individual checks though